src/screens/About/Team/TeamSection.tsx

```diff
@@ -38,6 +33,22 @@ const TeamSection = () => {
               })}
             </Grid>
           </Fragment>
+        ) : (
+          <Fragment key={team.teamName}>
+            <AboutHeading3>{team.teamName}</AboutHeading3>
+            <div>
+              {teamMembers
+                .map<React.ReactNode>(teamMember => {
+                  return (
+                    <AlumniMember
+                      teamMember={teamMember}
+                      key={teamMember.fullName}
+                    />
+                  );
+                })
+                .reduce((prev, curr) => [prev, ', ', curr])}
```

Comment:
wanna simplify and add the comma via css instead? ie adding this into the AlumniWrapper style block-

```
  &:not(:last-of-type){
    &:after {
      content: ', ';
    }
  }
```

up to you!

Reply:
🤯 great suggestion, thanks : )